build.gradle

```diff
@@ -722,6 +747,8 @@ project('controller') {
         compile group: 'org.glassfish.jersey.containers', name: 'jersey-container-grizzly2-http', version: jerseyVersion
         compile group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: jerseyVersion
         compile group: 'org.glassfish.jersey.media', name: 'jersey-media-json-jackson', version: jerseyVersion
+        compile group: 'com.fasterxml.jackson.jaxrs', name: 'jackson-jaxrs-json-provider', version: '2.9.10'
+        compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-joda', version: '2.9.10'
```

Comment:
These additions where necessary as a result of the changes that occurred in the [JacksonJsonProvider](controller/src/main/java/io/pravega/controller/server/rest/generated/api/JacksonJsonProvider.java).